apps/backend/app/services/fix_suggestion_engine.py

```
@@ -0,0 +1,669 @@
+
+        # Outlier capping is safe (doesn't remove rows)
+        if (fix.transformation_type == "outlier_removal" and
+                fix.parameters.get("method") == "cap"):
+            return True
+
+        return False
+
+    async def preview_fix(
+        self,
+        df: pd.DataFrame,
+        issue: DataIssue,
```

Ruff (0.14.8)

Unused method argument: `issue`

(ARG002)